Source/WebCore/Modules/applicationmanifest/ApplicationManifest.h

```diff
@@ -63,6 +63,7 @@ struct ApplicationManifest {
     URL startURL;
     Color themeColor;
     Vector<Icon> icons;
+    URL appId;
```

Comment:
NIT: I'd maybe move this to be closer to the `startURL` since the two are kinda intertwined

ditto below with all the other spots too